test/wasmBenchmarker/benchmark.py

```diff
@@ -143,14 +144,16 @@ def compile_tests(emcc_path, path, only_game=False, compile_anyway=False, simd=F
 
     return test_names
 
-def run_wasm(engine, path, name):
+def run_wasm(engine, path, name, jit=False):
     if not os.path.exists(path):
         print("invalid path for run")
         exit(1)
 
     tc_path = path + "/wasm/" + name + ".wasm"
     print("TC path: " + tc_path)
-    result = subprocess.check_output(engine + " --run-export runtime " + tc_path, shell=True)
+    flags = "--jit --run-export runtime" if jit else "--run-export runtime"
```

Comment:
This is a string duplication. Maybe this work:
`"--run-export runtime" + (" --jit" if jit else "")`